CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 ### Performance enhancements
 
 ### Bug fixes
+* `skbio.stats.composition.ancom` is compatible with scipy v0.18 and numpy v1.12.0 ([#1472](https://github.com/biocore/scikit-bio/issues/1472)).
```

Comment:
Would it be possible this to be more descriptive? As is, this reads like a new feature rather than a bug fix.